### PR TITLE
Additional remarks - Work Behavior Changes 781a

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Option to enter additional remarks in behavior - particularly work behavior changes.
These questions are asked after Additional Event screen and physical health and mental health additional remarks.
```bah-781a-additionalRemarks-work```
http://localhost:3001/disability-benefits/apply/form-526-all-claims/new-disabilities/ptsd-781a-work-changes
## Testing done

tests/pages/workBehaviorChanges.unit.spec.jsx

## Screenshots
screen shot 2018-11-16 at 7 22 22 am
screen shot 2018-11-16 at 7 22 08 am

## Acceptance criteria
- [ ] The PTSD type classification is correctly populated on page header based on the user's selection at the beginning of the workflow
- [ ] User is able to indicate changes to work habits as a result of the stressful incident per the Content Document
**Noted: No fields are required.**
## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]

**Review by jbalboni — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 1]
```diff
@@ -343,12 +345,20 @@ const formConfig = {
...
+          title: 'Additional Remarks - Mental Health Changes',
           path: 'new-disabilities/ptsd-781a-mental-changes',
           depends: isAnswering781aQuestions(0),
           uiSchema: mentalHealthChanges.uiSchema,
           schema: mentalHealthChanges.schema,
         },
+        workBehaviorChanges: {
+          title: 'Additional Remarks - Behavior Changes at Work',
+          path: 'new-disabilities/ptsd-781a-work-changes',
+          depends: formData =>
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/config/form.js]: Can we set this `depends` up just like the `mentalHealthChanges` and get rid of `isAnsweringPtsdForm` (in utils as well).

File: src/applications/disability-benefits/all-claims/utils.jsx [hunk 2]
```diff
@@ -591,6 +591,9 @@ export const getHomelessOrAtRisk = formData => {
 export const isNotUploadingPrivateMedical = formData =>
   _.get(DATA_PATHS.hasPrivateRecordsToUpload, formData) === false;
 
+export const isAnsweringPtsdForm = formData =>
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/utils.jsx]: delete

**Review by aaronjlech — CHANGES_REQUESTED** [review]

File: src/applications/disability-benefits/all-claims/config/form.js (hunk 1 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/config/form.js]: changed!

File: src/applications/disability-benefits/all-claims/utils.jsx (hunk 2 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/utils.jsx]: deleted!

**Review by aaronjlech — APPROVED** [review]

**Pull request merged**